### PR TITLE
Issue #15214: Migrated section 4.3 One statement per line to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214 

https://checkstyle.org/google_style.html#a4.3

>OneStatementPerLine

https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%2Fcom%2Fpuppycrawl%2Ftools%2Fcheckstyle%2Fchecks%2Fcoding+path%3A**%2Fmessages*.properties+repo%3Acheckstyle%2Fcheckstyle+%22multiple.statements.line%22&type=code

### CODE REVIEW

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java [hunk 1]
```diff
@@ -31,21 +34,35 @@ public class InputOneStatementPerLine {
   ;; // violation 'Only one statement per line allowed.'
   static {
     new JCheckBox().addActionListener((final ActionEvent e) -> { good(); });
+    // violation above ''{' at column 64 should have line break after.'
```
Comment (romani) [inline src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:37]: All we need in file for validation is `Only one statement per line allowed.`
Why you keep unrelated?
Let's resolve them in any way.

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java [hunk 2]
```diff
@@ -31,21 +34,35 @@ public class InputOneStatementPerLine {
...
     List<Integer> ints = new LinkedList<Integer>();
     ints.stream().map(t -> { return t * 2; }).filter(t -> { return false; });
+    // 2 violations above:
+    //  ''{' at column 28 should have line break after.'
+    //  ''{' at column 59 should have line break after.'
```
Comment (romani) [inline src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:42]: If no related violation messages, please resolve them.

**Review by romani — CHANGES_REQUESTED** [review]
Apply to all others

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:37]: I have intentionally keep these violations. They're helping to cover edge cases and show that such type of statements are allowed.

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java (hunk 2 above)
Comment (Zopsss) [reply src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:42]: reason behind violations: https://github.com/checkstyle/checkstyle/pull/15348#discussion_r1690157210

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java (hunk 1 above)
Comment (romani) [reply src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:37]: ok, lets keep it.

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java (hunk 2 above)
Comment (romani) [reply src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java:42]: ok, lets keep it.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**